dapr/aio/clients/grpc/client.py

```diff
@@ -1711,6 +1715,65 @@ async def purge_workflow(self, instance_id: str, workflow_component: str) -> Dap
         except grpc.aio.AioRpcError as err:
             raise DaprInternalError(err.details())
 
+    async def converse_alpha1(
+        self,
+        name: str,
+        inputs: List[ConversationInput],
+        *,
+        # Force remaining args to be keyword-only
+        context_id: Optional[str] = None,
+        parameters: Optional[Dict[str, GrpcAny]] = None,
+        metadata: Optional[Dict[str, str]] = None,
+        scrub_pii: Optional[bool] = None,
+        temperature: Optional[float] = None,
+    ) -> ConversationResponse:
+        """Invoke an LLM using the conversation API (Alpha).
+
+        Args:
+            name: Name of the LLM component to invoke
+            inputs: List of conversation inputs
+            context_id: Optional ID for continuing an existing chat
+            parameters: Optional custom parameters for the request
+            metadata: Optional metadata for the component
+            scrub_pii: Optional flag to scrub PII from outputs
```

Comment:
```suggestion
            scrub_pii: Optional flag to scrub PII from inputs and outputs
```
Scrubbing is done on both inputs and outputs:
https://github.com/dapr/dapr/blob/955436f45f783e52c9af8c2ae32f7f82a287c39c/pkg/api/universal/conversation.go#L67
https://github.com/dapr/dapr/blob/955436f45f783e52c9af8c2ae32f7f82a287c39c/pkg/api/universal/conversation.go#L116